### DISCUSSION TITLE
3.1.5版本 关于shell节点运行失败如何才能继续运行下一个节点

### DISCUSSION THREAD

**Discussion by Hannibal0828** [apache/dolphinscheduler#15050] [Q&A]
创建一个工作流定义，有两个shell节点且为串行，运行节点一失败，然后整个工作流任务就失败了，不会运行节点二，请问如何操作可以在前一个节点失败的情况下继续运行下一个节点。
![image](https://github.com/apache/dolphinscheduler/assets/54615461/46fd25a3-7006-45e2-8d2d-7bc55baa3bb7)
![image](https://github.com/apache/dolphinscheduler/assets/54615461/4f31ce34-81d1-4c08-a9f8-ae20dc08ecdf)

**Comment by zui01** [comment]
没办法
